stage1/init/init.go

```diff
@@ -483,13 +467,13 @@ func stage1() int {
 		}
 	}
 
-	flavor, systemdStage1Version, err := p.getFlavor()
+	flavor, _, err := p.getFlavor()
```

Comment:
Shouldn't this commit modify the `getFlavor` to simply not return the systemd version anymore? Maybe we could get rid of systemd versioning from build system? Or it this too far-fetched for now?